diff --git a/node_modules/jest-environment-miniflare/dist/src/index.js b/node_modules/jest-environment-miniflare/dist/src/index.js
index cd62922..bbddaf9 100644
--- a/node_modules/jest-environment-miniflare/dist/src/index.js
+++ b/node_modules/jest-environment-miniflare/dist/src/index.js
@@ -76,6 +76,7 @@ var MiniflareEnvironment = class {
     const global = this.global = vm.runInContext("this", this.context);
     global.global = global;
     global.self = global;
+    global.Buffer = Buffer;
     global.clearInterval = clearInterval;
     global.clearTimeout = clearTimeout;
     global.setInterval = setInterval;
